akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -350,14 +350,19 @@ object Attributes {
    * Configures `log()` operator log-levels to be used when logging.
    * Logging a certain operation can be completely disabled by using [[LogLevels.Off]].
    *
-   * Passing in null as any of the arguments sets the level to its default value, which is:
-   * `Debug` for `onElement` and `onFinish`, and `Error` for `onFailure`.
    */
   def createLogLevels(onElement: Logging.LogLevel, onFinish: Logging.LogLevel, onFailure: Logging.LogLevel): Attributes =
-    logLevels(
-      onElement = Option(onElement).getOrElse(Logging.DebugLevel),
-      onFinish = Option(onFinish).getOrElse(Logging.DebugLevel),
-      onFailure = Option(onFailure).getOrElse(Logging.ErrorLevel))
+    logLevels(onElement, onFinish, onFailure)
+
+  /**
+   * Java API
+   *
+   * Configures `log()` operator log-levels to be used when logging onElement.
+   * Logging a certain operation can be completely disabled by using [[LogLevels.Off]].
+   *
+   */
+  def createLogLevels(onElement: Logging.LogLevel): Attributes =
+    logLevels(onElement)
 
   /**
    * Configures `log()` operator log-levels to be used when logging.
```

Comment:
Next line should be `[[LogLevels.Off]]`

Reply:
@patriknw check now, i think all of them are fixed, hope I didn't miss anything ;-)

Reply:
Looks good